index.rst

```diff
@@ -12,18 +12,633 @@
+
+
+In the following figure we display the scatter of transient candidates.
+
+.. figure:: /_static/figures/chipillum_scatter_diffim_flavors.png
+    :name: chipillum_scatter_diffim_flavors
+    :target: ../_images/chipillum_scatter_diffim_flavors.png
+    :alt: Location of the detected sources in the CCD chip.
+
+    The location of the detected transient candidates in the CCD chip. We call this the "illumination" of the CCD. It is clear again, an excess of detections in the central pixel columns and the chip edges.
+
+The figure shows that there are columns with detection excess that can be removed, by using different flags or using information in the image mask plane. In the following figure we can see the scatter of the sources that pass the flag cuts.
+
+.. figure:: /_static/figures/chipillum_scatter_goodsrc_diffim_flavors.png
+    :name: chipillum_scatter_goodsrc_diffim_flavors
+    :target: ../_images/chipillum_scatter_goodsrc_diffim_flavors.png
+    :alt: Location of the detected sources in the CCD chip after common flag cuts.
+
+    Position of the detected transient candidates that pass flag cuts in the CCD chip.
+
+The flags cut the excess of detections up to certain level, and also leave areas in the CCD where now we have an artificial under-density of detections.
+
+.. figure:: /_static/figures/chipillum_x_y_histogram_goodsrcs_diffim_flavors.png
+    :name: chipillum_x_y_histogram_goodsrcs_diffim_flavors
+    :target: ../_images/chipillum_x_y_histogram_goodsrcs_diffim_flavors.png
+    :alt: X-Y pixel coordinate distributions for diaSources that pass flag cuts
+
+    Distribution of the X and Y coordinates of the diaSources that pass the conventional flag cuts.
+
+In the distribution of coordinates we can see that the edge excess and central excess get flattened and apparently the artifact sources are cut away.
+
+We can study again the location distribution, but as a function to the distance to the center of the image, however since the image is rectangular this will not yield a flat distribution.
+Instead we can weight each source detection by the area of the smallest central rectangle that contains the detection. This means, taking each pair of transient coordinate :math:`(x,y)` and transforming it to obtain the side of this rectangle: :math:`l = max(x', y')` with :math:`(x', y') = (x-x_c, y-y_c)`. The value :math:`\gamma` that we take finally into account is the fractional area with respect to the full CCD area: :math:`\gamma = l^2/(4096 \times 2048)`
+This quantity will be distributed as a random uniform distribution for a true random position of our
+
+.. figure:: /_static/figures/chipillum_areafraction_combined_diffim_flavors.png
```

Comment:
This figure and the next one both appear slightly cropped in the final index.rst I'm reading. The x-axis label is mostly unreadable.

Reply:
Yes, you are right.